app/models/admin.rb

```diff
@@ -0,0 +1,7 @@
+class Admin < ApplicationRecord
```

Comment:
probably you are not doing anything with this model, as I saw you are signing in and out the user model.
but with admin model, it could be less tricky, later you will need to setup this admin model to login to admin panel